pages/common/pprof.md

```diff
@@ -0,0 +1,24 @@
+# pprof
+
+> Command-line tool for visualization and analysis of profile data.
+> More information: <https://github.com/google/pprof>.
+
+- Generate a text report from `fibbo-profile.pb.gz` file, on fibbo binary:
+
+`pprof -top {{./fibbo}} {{./fibbo-profile.pb.gz}}`
+
+- Generate a graph from fibbo-profile.pb.gz profile data and open it on a web browser:
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a graph and open it on a web browser:
```

We've already used similar language in the previous example, so I'm not sure we need to repeat ourselves here.